ntp-daemon/src/main.rs

```diff
@@ -23,5 +27,66 @@ async fn main() -> Result<(), Box<dyn Error>> {
     // this away when not using sentry.
     let _guard = finish_tracing_init(&mut config, has_log_override)?;
 
-    ntp_daemon::spawn(&config.system, &config.peers).await
+    let peers_reader = Arc::new(tokio::sync::RwLock::new(Vec::new()));
+    let peers_writer = peers_reader.clone();
+
+    std::fs::remove_dir_all("/run/ntpd-rs")?;
+    std::fs::create_dir("/run/ntpd-rs")?;
+
+    let peers_listener = UnixListener::bind("/run/ntpd-rs/observe")?;
```

Comment:
we can make this path configurable, though that does mean that the client should use the same config.

Reply:
this path should be configurable. Client can accept the path on the commandline or via configuration if it is non-default.